drivers/adc/ad463x/iio_ad463x.h

```diff
@@ -62,7 +62,8 @@ struct iio_ad463x {
 	struct ad463x_dev *ad463x_desc;
 };
 
-extern struct iio_device ad463x_iio_desc;
+extern struct iio_device ad463x_iio_desc_one_chn;
```

Comment:
why do these need to be extern?

Reply:
Also don't know, just kept as it was before. but i will remove